src/main/java/com/google/firebase/auth/UserRecord.java

```diff
@@ -193,9 +193,10 @@ public boolean isDisabled() {
   }
 
   /**
-   * Returns the timestamp beginning with which tokens are valid in seconds since the epoch.
+   * Returns the timestamp beginning with which tokens are valid in milliseconds since the epoch.
```

Comment:
Returns a timestamp in milliseconds since epoch, truncated down to the closest second. Tokens minted before this timestamp are considered invalid.